js/index.js

```diff
@@ -1,7 +1,114 @@
 // Iteration 1: Names and Input
+var hacker1 = "Daniel Hamekasi";
+console.log(`"The driver's name is ${hacker1}"`);
 
+var hacker2 = "John Doe";
+console.log(`"The navigator's name is ${hacker2}"`);
 
 // Iteration 2: Conditionals
 
+if (hacker1.length > hacker2.length) {
+  console.log(
+    `The driver has the longest name, it has ${hacker1.length} characters.`
+  );
+} else if (hacker1.length < hacker2.length) {
+  console.log(
+    `It seems that the navigator has the longest name, it has ${hacker2.length} characters.`
+  );
+} else {
+  console.log(
+    `Wow, you both have equally long names, ${hacker1.length} characters!`
+  );
+}
 
-// Iteration 3: Loops
\ No newline at end of file
+// Iteration 3: Loops
+
+//3.1
+var hacker1Str = ``;
+for (var i = 0; i < hacker1.length - 1; i++) {
+  hacker1Str += hacker1[i].toUpperCase() + " ";
+}
+hacker1Str += hacker1[i].toUpperCase();
+console.log(hacker1Str);
+
+//3.2
+var hacker2Str = ``;
+for (var i = hacker2.length - 1; i >= 0; i--) {
+  hacker2Str += hacker2[i];
+}
+console.log(hacker2Str);
+
+//3.3 Didnt know if you expected us to use nested for loops? found this method instead, although maybe that wasnt the correct approach.
```

Comment:
Good that you found this method, you'll get to use it quite often.
You could have used a for loop also, nesting the conditions inside the loop and comparing each character like Uros showed this morning (don't hesitate if you want to review it).